python/adapters/postgres_storage_adapter.py

```diff
@@ -53,78 +43,57 @@ def _commit(self):
     def _rollback(self):
         self.conn.rollback()
 
-    def convert_payment(self, source, destination, amount, memo, tx_fee, tx_charged_fee, op_index, tx_status, op_status,
-                        tx_hash, timestamp):
-        payment = dict.fromkeys(self.payments_output_schema())
-        payment['source'] = source
-        payment['destination'] = destination
-        payment['amount'] = amount
-        payment['memo_text'] = memo
-        payment['fee'] = tx_fee
-        payment['fee_charged'] = tx_charged_fee
-        payment['operation_index'] = op_index
-        payment['tx_status'] = tx_status
-        payment['op_status'] = op_status
-        payment['hash'] = tx_hash
-        payment['time'] = datetime.utcfromtimestamp(timestamp)
-
-        return payment
-
-    def convert_creation(self, source, destination, balance, memo, tx_fee, tx_charged_fee, op_index, tx_status,
-                         op_status, tx_hash, timestamp):
-        creation = dict.fromkeys(self.creations_output_schema())
-        creation['source'] = source
-        creation['destination'] = destination
-        creation['starting_balance'] = balance
-        creation['memo_text'] = memo
-        creation['fee'] = tx_fee
-        creation['fee_charged'] = tx_charged_fee
-        creation['operation_index'] = op_index
-        creation['tx_status'] = tx_status
-        creation['op_status'] = op_status
-        creation['hash'] = tx_hash
-        creation['time'] = datetime.utcfromtimestamp(timestamp)
-
-        return creation
+    def convert_operation(self, source, destination, amount, tx_order, tx_memo, tx_account, tx_account_sequence,
+                          tx_fee, tx_charged_fee, tx_status, tx_hash, op_order, op_status, op_type, timestamp,
+                          is_signed_by_app, ledger_file_name, ledger_sequence):
+
+        operation = dict.fromkeys(self.operation_output_schema())
```

Comment:
why is this needed if you override everything?

Reply:
To guarantee that all of the keys exists based on the output schema, even if they are empty, or any future assignment will be missed, but you next comment could prevent that